app/src/main/java/com/sdp/movemeet/view/chat/ChatActivity.java

```diff
@@ -150,27 +149,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
     }
 
-    public void createDrawer() {
-        drawerLayout = findViewById(R.id.drawer_layout);
-        navigationView = findViewById(R.id.nav_view);
-        textView = findViewById(R.id.textView);
-        toolbar = findViewById(R.id.toolbar);
-
-        navigationView.bringToFront();
-        ActionBarDrawerToggle toggle = new
-                ActionBarDrawerToggle(this, drawerLayout, toolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close);
-        drawerLayout.addDrawerListener(toggle);
-
-        View hView = navigationView.inflateHeaderView(R.layout.header);
-
-        fullNameDrawer = hView.findViewById(R.id.text_view_profile_name);
-        phoneDrawer = hView.findViewById(R.id.text_view_profile_phone);
-        emailDrawer = hView.findViewById(R.id.text_view_profile_email);
-
-        toggle.syncState();
-        navigationView.setNavigationItemSelectedListener(this::onNavigationItemSelected);
-        navigationView.setCheckedItem(R.id.nav_chat);
-    }
```

Comment:
Being able to remove all those lines in each file will allow us to increase our code coverage in the end! That's a good point!

Reply:
That's right ;)